sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -560,6 +570,36 @@ private <T> Mono<CosmosItemResponse<T>> createItemInternal(T item, CosmosItemReq
                 trackingId);
     }
 
+    private void applyPoliciesAndGetRequestOptions(OperationType operationType, ResourceType resourceType, OverridableRequestOptions requestOptions,
+                                                             String spanName) {
+        CosmosAsyncClient client = this.database.getClient();
```

Comment:
Are there other parts of the CosmosDiagnosticContext that are relevant to show to the customer?